### PR TITLE
[Docs] Updated url of images in docs

### PR DESCRIPTION
**Pull request by AakankshaSanctis**
Changes proposed in this pull request:
- Url of images with links to wiki repository

## Notify reviewers
@prasadtalasila

### CODE REVIEW

File: analysis_techniques.html [hunk 1]
```diff
@@ -18,6 +18,7 @@
         SOURCELINK_SUFFIX: '.txt'
       };
     </script>
+
```
Comment (prasadtalasila) [inline analysis_techniques.html]: This newline is not required

File: analysis_techniques.html [hunk 2]
```diff
@@ -31,7 +32,8 @@
     <div class="related" role="navigation" aria-label="related navigation">
       <h3>Navigation</h3>
       <ul>
-        <li class="right" style="margin-right: 10px">
+        <li class="ri
```
Comment (prasadtalasila) [inline analysis_techniques.html]: This change is not required.

**Review by prasadtalasila — CHANGES_REQUESTED** [review]
@AakankshaSanctis please see the suggested reverts

**Comment by AakankshaSanctis** [comment]
Changes have been made

**Comment by prasadtalasila** [comment]
@AakankshaSanctis I did not check before, but this PR is for gh-pages branch. The gh-pages branch is an autogenerated documentation branch. It is not to be manually modified.

You need to check the corresponding sphynx documentation and modify them. Once modified, the deploy_docs.sh script would automatically generate the correct html pages.

Please fork the dev branch, rename the branch, modify sphynx files and open a PR to dev branch.

**Pull request closed without merging**